doc/ProofGeneral.texi

```diff
@@ -5135,6 +5191,41 @@ General will perform some unnecessary compilations.
 @end itemize
 
 
+@node Omitting proofs for speed
+@section Omitting proofs for speed
+
+To speed up asserting larger chunks, Proof General can omit
+complete opaque proofs by silently replacing the whole proof
+script with @code{Admitted}, @ref{Script processing commands}.
+This works when
+@itemize
+@item
+proofs are not nested
+@item
+opaque and non-opaque proofs start with @code{Proof.} or
+@code{Proof using}
```

Comment:
@hendriktews BTW I believe it could be worth it to add in the manual, a comment/disclaimer/advice saying that:

> If the proof script relies on sections, it is highly recommended to use a `Proof using` clause for each lemmas contained in the sections, so that the Admitted lemmas are generalized in an accurate way. For completeness, note that PG provides a dedicated feature to generate these `Proof using` clauses automatically (a defective form being e.g. `Proof using Type` if no hypothesis is used): see the menu command `Coq > "Proof using" mode`…

(but feel free to amend the prose!)

Reply:
There is already Section 11.3 Proof using annotations, which recommends `Proof using`. Moreover, a `Proof using` recommendation has not much to do with this PR. May I therefore suggest that you check Section 11.3 and amend it, if you believe it is not sufficient?